packages/react/src/internal/indicator-container/splitContainerProps.ts

```diff
@@ -0,0 +1,12 @@
+import { partitionObject } from '../../utils';
+
+/**
+ * Splits `Indicator` properties into `label` and `input` props.
+ */
+export const splitContainerProps = (props: Input | Label) =>
+  partitionObject(props, ([key]) => containerProps.test(key));
+
+const containerProps = /^aria-|^data-/;
+
+type Input = JSX.IntrinsicElements['input'];
+type Label = JSX.IntrinsicElements['label'];
```

Comment:
In other places we have different naming for element props, what do you think about:

```suggestion
export const splitContainerProps = (props: InputElementProps | LabelElementProps) =>
  partitionObject(props, ([key]) => containerProps.test(key));

const containerProps = /^aria-|^data-/;

type InputElementProps = JSX.IntrinsicElements['input'];
type LabelElementProps = JSX.IntrinsicElements['label'];
```

Reply:
Yup I'll keep the consistency, even though the verbosity drives me nuts 😅